docs/build.sh

```diff
@@ -11,13 +11,5 @@
 cd $(dirname "$0")
 cd ..
 
-sphinx=sphinx-build
-if [ -n "$SPHINX" ]; then
-    sphinx=$SPHINX
-fi
-
-if [ -z "$JOBS" ]; then
-    JOBS=2
-fi
-
-"$sphinx" -a -b html . ./docs/html -w ./docs/_sphinx-warnings.txt -j "$JOBS" "$@"
+"${SPHINX:-sphinx-build}" -E -b html -d build/docs/html . docs/html -w build/docs/html/_sphinx-warnings.txt -j "${JOBS:-auto}" "$@"
```

Comment:
I would like to leave `-E` out if possible because it makes incremental builds much slower. Could you give some more details on the issue in fdd406b7220d6c2086a959a2c6cd69a5c7858644?